hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -90,7 +90,7 @@ private void handleGetClusterVersion(HttpGetCommand command) {
 
     private void handleCluster(HttpGetCommand command) {
         Node node = textCommandService.getNode();
-        StringBuilder res = new StringBuilder(node.getClusterService().membersString());
+        StringBuilder res = new StringBuilder(node.getClusterService().memberListString());
```

Comment:
The string format in the response to will be always the new one, no matter the value of `hazelcast.legacy.memberlist.format.enabled`, right?